applications/ExaquteSandboxApplication/tests/run_cpp_unit_tests.py

```diff
@@ -1,5 +1,6 @@
 from KratosMultiphysics import *
 from KratosMultiphysics.ExaquteSandboxApplication import *
+from KratosMultiphysics.MeshingApplication import *
```

Comment:
You should add a check and a try, otherwise these tests will not run without MeshingCompiled

Reply:
I added a check in #5147 [here](https://github.com/KratosMultiphysics/Kratos/pull/5147#discussion_r299012076) to see if Meshing Application is compiled. In any case I also add it here just for safety.

Reply:
Done